.github/workflows/publish.yml

```diff
@@ -0,0 +1,80 @@
+name: "Publish"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-github-release:
+    name: Deploy GitHub Release
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
is there a specific reason for using v2 here rather than v3? The refactoring you've done above has added v3